### ISSUE TITLE
Start month

### ISSUE THREAD

**Issue by steunix** [bendavis78/paper-date-picker#8]
Hi, excellent work, first of all. I just suggest that the initial month of the calendar should be the one of the initial date: starting always from January is mostly impractical

**Comment by bendavis78** [comment]
Hmm, it should automatically scroll to the selected month on open. That's a bug.

**Comment by suexID** [comment]
We're experiencing this too. :)

**Comment by gulyasfeccferenc** [comment]
I got the same problem. Hope you find tha way to eliminate this issue soon

**Comment by gulyasfeccferenc** [comment]
Woohoo! Thanks for the quick response. After your fix, its work now.

**Comment by jasongardnerlv** [comment]
Seems that it ignores the scrollTo as it is not sized yet (offsetHeight = 0).  I worked around for now with:
```
_showDialog: function() {
  this.$.dialog.toggle();
  this.async(function() {
    this.$.picker.$.calendar.updateSize();
  }.bind(this));
},
```

**Comment by JaySunSyn** [comment]
samer here.

**Comment by Compufreak345** [comment]
@jasongardnerlv Thanks, that helped! Unfortunately it scrolled to August whereas the selected date was July :0 (I use de locale if it makes any difference)

**Comment by bendavis78** [comment]
This should have been fixed since 1.0.3. Are you still seeing an issue? If so, can you provide steps to reproduce?

**Comment by bendavis78** [comment]
Closing for now. Let me know if anyone's still experiencing issues.
